extensions/cornerstone-dicom-pmap/src/utils/initPMAPToolGroup.ts

```diff
@@ -0,0 +1,6 @@
+function createPMAPToolGroupAndAddTools(ToolGroupService, customizationService, toolGroupId) {
+  const { tools } = customizationService.get('cornerstone.overlayViewportTools') ?? {};
```

Comment:
I'm not sure what overlayViewportTools is, but it seems like someone else added it rather than you.